src/util/markdown-input.html

```diff
@@ -73,8 +73,7 @@
       paper-icon-button {
         margin-top: 1em;
       }
-      paper-icon-button:hover,
-      paper-icon-button[active] {
```

Comment:
Know we switch from view to draw and therefore the draw icon would always be blue.